lib/search/query.rb

```diff
@@ -94,7 +94,7 @@ def rerank(es_response, search_params)
 
       return { reranked: false, es_response: es_response } if reranked.nil?
 
-      es_response["hits"]["hits"] = reranked
+      es_response["hits"]["hits"] = reranked.first(search_params.count)
```

Comment:
If `k` is `2 * count` then might we need to flipflop between first(count) and last(count) depending if `start % k` is 0 or `count`? We could end up reranking results out of existence otherwise?

Reply:
We could introduce some jitter here to give some results a chance. Maybe @hannako has some ideas on how we might occassionally swap downranked results into the top-k. 🤔 

We could ocassionally return the bottom-k to some users, might be fairer to spread it out a bit among users, since we can assume most of the time the top-k is going to be better.